server/sonar-web/src/main/js/apps/update-center/plugins.js

```diff
@@ -40,7 +40,7 @@ const Plugins = Backbone.Collection.extend({
     const that = this;
     return r.plugins.map(plugin => {
       let updates = [
-        that._getLastWithStatus(plugin.updates, 'COMPATIBLE'),
+        that._getLastWithStatus(plugin.updates, 'INCOMPATIBLE'),
```

Comment:
what's this?

Reply:
:expressionless: oups, something I forgot to remove